.github/workflows/ci-scripts-build.yml

```diff
@@ -22,6 +22,7 @@ env:
     SETUP_PATH: .ci-local:.ci
     BASE: SELF
     EPICS_TEST_IMPRECISE_TIMING: YES
+    EPICS_TEST_TIMEOUT: 1200 # 20 min (RTEMS is slowest)
```

Comment:
RTEMS may take 20 minutes to run all the tests, but this implies there's a specific test that needs 20 minutes, which can't be true.